testsuite/tests/typing-polyvariants-bugs/pr10664.reference~

```diff
@@ -1 +0,0 @@
-A
```

Comment:
Why this diff ?

Reply:
This looks like a vim backup file that was mistakenly committed in https://github.com/ocaml/ocaml/commit/ad82f362a6df4ff8af51b996b5a7399fa7eb01e6.